glossary.md

```diff
@@ -39,3 +39,19 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **Measurement method**
+
+The source of and technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach, and names the group making the measurement. Examples include `BosniaCensus` and `WorldHealthOrganizationEstimates`.
```

Comment:
It's not really meant to be the source, just how a statistical variable was measured. The how just often happens to be source dependent. E.g. an Anne estimate may not = a Bob estimate.

Reply:
Let's try to leave out source here